### PR TITLE
Merging Student Profiles and Data Vis

### PR DESCRIPTION
**Pull request by martinchoffman**

### CODE REVIEW

File: app/backend/key/models.py [hunk 1]
```diff
@@ -104,6 +116,11 @@ class Students(models.Model):
     id = models.IntegerField(blank=True, primary_key=True)
     first_attendance = models.DateField(blank=True, null=True)
     number_visits = models.IntegerField(blank=True, null=True)
+    birthday = models.DateField(blank=True, null=True)
+    nickname = models.TextField(blank=True, null=True)
+    gender = models.TextField(blank=True, null=True)
+    student_id = models.TextField(blank=True, null=True)
+    #photo = models.ImageField(blank=True, null=True, upload_to='profile_photos/')
```
Comment (marasybesma) [inline app/backend/key/models.py:123]: is this something we're implementing later?

File: notes_for_jordan.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Basically I want django to count how many records there are per unique date - rounded hour combination.
+
+** right now, to check the output of this reports/byHourAttendance?startdate=xxxxxx&enddate=xxxx , you can go to the reports tab of our site and it's being printed in the inspect console (thru react reports view).
+
+
+Possible paths to a solution:
+https://stackoverflow.com/questions/30465013/django-group-by-hour
+(above requires date and time combined into one field though)
+
+https://stackoverflow.com/questions/17682567/how-to-add-a-calculated-field-to-a-django-model
```
Comment (marasybesma) [inline notes_for_jordan.txt:21]: we can probably delete this file now

**Review by marasybesma — APPROVED** [review]
looks good - although 'notes_for_jordan.txt' probably doesnt need to be in the repo anymore.

**Comment by musicalforks** [comment]
Looks good to me! Yay!

**Pull request merged**